packages/client/src/modals/Transaction/BannerInfo.js

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useContext, useState } from "react";
+import { Web3Context } from "contexts/Web3";
+import { ACTION_TYPES } from "constants/enums";
+import Svg from "library/Svg";
+import { formatAddress, getTokenMeta, parseIdentifier, getNameByAddress } from "utils";
+
+const BannerInfo = ({ className = "", actionData = {}, contacts = {}, signers = [] }) => {
+  const { getAccountNFTReference } = useContext(Web3Context);
+  const [image, setImage] = useState();
+  const { recipient, nftId, collectionId, vaultId, tokenAmount, signerAddr } = actionData;
+  const { displayName, tokenType } = getTokenMeta(vaultId);
+  const { contractName: NFTName, contractAddress: NFTAddress } = parseIdentifier(collectionId);
+  const { name: imageName, imageURI } = image || {};

```

Comment:
Suggest moving this part to line 9 on initializing the state so the structure is clearer
```suggestion
  const [image, setImage] = useState({
      imageName:undefined,
      imageURI:undefined
  })

```